styles/fonts.css

```
@@ -0,0 +1,5 @@
+@theme {
+  --font-sans: var(--font-esamanru), -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.5.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)